docs/deploy-full-rp-service-in-dev.md

```diff
@@ -0,0 +1,108 @@
+# Deploy an Entire RP Development Service
+
+## Prerequisites
+
+1. Your development environment is prepared according to the steps outlined in [Prepare Your Dev Environment](./prepare-your-dev-environment.md)
+
+
+## Deploying an int-like Development RP
+
+1. Fetch the most up-to-date secrets with `make secrets`
+
+1. Copy and source your environment file.
+    ```bash
+    cp env.example env
+    vi env
+    . ./env
+    ```
+
+1. Generate the development RP configuration
+    ```bash
+    go run ./hack/genconfig > ./dev-config.yaml
+    ```
+
+1. Set environment variables based on the generated configuration.
+    ```bash
+    export KEYVAULT_PREFIX=FILL_IN   # rps[0].configuration.keyvaultPrefix
+    export DST_ACR_NAME=FILL_IN      # the prefix on '.azurecr.io' named 'rpImagePrefix' in dev-config.yaml
+    export RP_RESOURCE_GROUP=FILL_IN # rps[0].rpResourceGroupName
+
+    export ARO_IMAGE=${DST_ACR_NAME}.azurecr.io/aro:$(git rev-parse --short HEAD)-dirty
+    ```
+
+1. Run `make deploy`
+> __NOTE:__ This will fail on the first attempt to run due to certificate and container mirroring requirements.
+
+1. Update the certificates in keyvault
+    ```bash
+    az keyvault certificate import \
```

Comment:
can we avoid duplicating this list somehow?  Not sure how.

Reply:
Not sure how either.  There are a few different spots in our code where we have similar, but not exactly the same.  I'm going to leave it how it is for now, can revisit this later unless someone has a better idea.